src/pages/SingleArticlePage.jsx

```diff
@@ -1,26 +1,31 @@
 import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
-import { getArticleById } from "../utils/fetch";
+import { getArticleById, getCommentByArticle } from "../utils/fetch";
 import SingleArticle from "../components/SingleArticle";
 import SingleArticleID from "../components/SingleArticleID";
+import CommentList from "../components/CommentList";
 
 const SingleArticlePage = () => {
   const [singleArticle, setSingleArticle] = useState([]);
+  const [comments, setComments] = useState([])
   const [isLoading, setIsLoading] = useState(true);
   const { id } = useParams();
-  console.log(id);
   useEffect(() => {
     getArticleById(id).then((response) => {
       setSingleArticle(response.data)
       setIsLoading(false);
     });
+    getCommentByArticle(id).then((response) => {
+        setComments(response.data)
+    })
   }, []);
```

Comment:
You could extract your loading state to prevent the single article page from becoming bloated.